source/FreeRTOS_ARP.c

```diff
@@ -328,8 +328,8 @@ static void vProcessARPPacketReply( const ARPPacket_t * pxARPFrame,
 
     if( pxARPWaitingNetworkBuffer != NULL )
     {
-        /* MISRA C-2012 Rule 11.3 warns about casting pointer type to a different data type.
-         * The struct to be casted to is defined as a packed struct.  The cast won't cause misalignment. */
+        /* MISRA Ref 11.3.1 [Misaligned access and packed structures] */
+/* More details at: https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/main/MISRA.md#rule-113 */
```

Comment:
Using a "//" inside of a "/*" maybe a violation of MISRA rule 3.1 - "character sequences /* and // shall not be used within a comment". The permitted exception here says use // within a //.

Would the MISRA.md URL render into a hyperlink when not viewed in a browser? I guess this depends upon the editor/IDE being used. 
The original thought was to have just a "MISRA.md" link to the target URL/file, but realise now that this may not be possible, as i'm not sure mark down is possible within comments.

Reply:
Yes, this will violate a MISRA rule.
I think we should suppress that rule globally anyways since we use a lot of links in the source code.